src/styles/variables.scss

```diff
@@ -4,6 +4,10 @@
 
 $white: #fff !default;
 
+// Interim color variable for migration purposes.
```

Comment:
I'm not thrilled here to have another grey, but this color of grey is not substitute-able with `$grey-1` as it is quite subtle.